feature-group-definitions/aria-attribute-reflection.dist.yml

```diff
@@ -0,0 +1,96 @@
+# Generated from: aria-attribute-reflection.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: ARIA attribute reflection (initial support)
+description: WAI-ARIA attributes have corresponding properties on `Element` and `ElementInternals` objects.
+spec: https://w3c.github.io/aria/#accessibilityroleandproperties-correspondence
+status:
+  baseline: low
+  baseline_low_date: 2023-10-24
+  support:
+    chrome: "103"
```

Comment:
https://chromiumdash.appspot.com/commit/dbef8d4838807211e2940f0929c8109cdf06b9d5 is the initial support for reflected ARIA properties in Chromium, and includes most of this stuff.

Can you exclude `ariaBrailleLabel`, `ariaBrailleRoleDescription`, `ariaDescription`, `ariaInvalid`, and `ariaSetSize` and see if that results in Chrome 81?

Reply:
We talked about this more on our call the other week. The minimalist set consisting of `role` and `ariaLabel` is still Chrome 103, thanks to `role`. As a consequence, I've added back everything that gets us down to Safari 12.1 (which is basically ElementInternals, plausibly a separate feature). Done with 6ab45161c3852b74d0240ca2847ddf64cd10cf10.